.github/workflows/primer_run_pr.yaml

```diff
@@ -33,12 +33,14 @@ permissions:
 
 jobs:
   run-primer:
-    name: Run / ${{ matrix.python-version }}
+    name: Run / ${{ matrix.python-version }} / batch index ${{ matrix.batchIdx }}
     runs-on: ubuntu-latest
-    timeout-minutes: 120
+    timeout-minutes: 45
     strategy:
       matrix:
         python-version: ["3.8", "3.11"]
+        batches: [4]
```

Comment:
Guess we can't do `len(batchIdx)` or something like that?

Reply:
Yeah, unless we add a step to use bash or python to do math, I don't think I can do it inside a string formatting expression.